### ISSUE TITLE
Change of defaults for NS 7.5

### ISSUE THREAD

**Issue by DavidePrincipi** [NethServer/dev#5490]
We planned the modification of some DB default values and Software center module compositions, starting from the release of NethServer 7.5.

Default DB value changes

- [x] TLS policy `20180330`
- [x] Server Manager web session expiry, idle 15 minutes, lifetime 8 hours
- [x] Windows file server page `Grant full control to the creator`
- [x] Remove fstab management
- [x] Rebuild nut to avoid [upstream bug](https://bugzilla.redhat.com/show_bug.cgi?id=1570146)
- [x] Set `postfix/MxRecordStatus=disabled`
- [x] Avoid Shorewall warning

Software center (comps) changes

- [x] mail2 (rspamd) replaces amavisd in Software Center
- [x] nethserver-subscription in ISO group
- [x] Remove WebVirtMgr section
- [x] Remove nethserver-libvirt package

Docs updates:

- [x] Merge Email 2 into Email section
- [x] Remove WebVirtMgr section
- [x] Update [manual for fstab](https://github.com/NethServer/nethserver-base/pull/111#issuecomment-388074457)

**See also**

- https://community.nethserver.org/t/its-time-to-test-centos-7-5/9743/16
- https://wiki.centos.org/Manuals/ReleaseNotes/CentOS7.1804?action=show&redirect=Manuals%2FReleaseNotes%2FCentOS7

**Comment by stephdl** [comment]
nethserver-fail2ban ?
see https://github.com/NethServer/comps/pull/11

**Comment by DavidePrincipi** [comment]
It's tracked by another issue (and not blocking/blocked by 7.5), I think it does not belong here.

**Comment by DavidePrincipi** [comment]
The `nethserver-subscription-ui` was added to ISO comps group with https://github.com/NethServer/comps/pull/6

**Comment by DavidePrincipi** [comment]
Added docs updates:

- Merge Email 2 into Email section
- Remove WebVirtMgr section

**Comment by nethbot** [comment]
in `7.5.1804/testing`:
- [nethserver-base-3.1.5-1.2.gf4ef4ee.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/testing/x86_64/Packages/nethserver-base-3.1.5-1.2.gf4ef4ee.ns7.noarch.rpm)

**Comment by nethbot** [comment]
in `7.5.1804/testing`:
- [nethserver-base-3.1.5-1.4.gba58ecc.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/testing/x86_64/Packages/nethserver-base-3.1.5-1.4.gba58ecc.ns7.noarch.rpm)

**Comment by nethbot** [comment]
in `7.5.1804/testing`:
- [nethserver-samba-4.0.1-1.2.g8777d61.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/testing/x86_64/Packages/nethserver-samba-4.0.1-1.2.g8777d61.ns7.noarch.rpm)

**Comment by nethbot** [comment]
in `7.5.1804/testing`:
- [nethserver-httpd-admin-2.2.1-1.2.g62e3f8a.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/testing/x86_64/Packages/nethserver-httpd-admin-2.2.1-1.2.g62e3f8a.ns7.noarch.rpm)

**Comment by DavidePrincipi** [comment]
What about `postfix/MxRecordStatus=disabled`? 

The additional DNS records do not work with Letsencrypt certs and Kerberos / GSSAPI authentication.

**Comment by nethbot** [comment]
in `7.5.1804/testing`:
- [nethserver-samba-4.0.1-1.4.g4d198ca.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/testing/x86_64/Packages/nethserver-samba-4.0.1-1.4.g4d198ca.ns7.noarch.rpm)

**Comment by nethbot** [comment]
in `7.5.1804/testing`:
- [nethserver-firewall-base-3.3.2-1.19.g14ca8d7.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/testing/x86_64/Packages/nethserver-firewall-base-3.3.2-1.19.g14ca8d7.ns7.noarch.rpm)
- [nethserver-firewall-base-ui-3.3.2-1.19.g14ca8d7.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/testing/x86_64/Packages/nethserver-firewall-base-ui-3.3.2-1.19.g14ca8d7.ns7.noarch.rpm)

**Comment by DavidePrincipi** [comment]
Set postfix/MxRecordStatus=disabled

**Comment by gsanchietti** [comment]
In `7.5.1804/base`:
- nut-2.7.4-17.ns7.x86_64.rpm 
￼- nut-cgi-2.7.4-17.ns7.x86_64.rpm 
- nut-client-2.7.4-17.ns7.x86_64.rpm	 
- nut-debuginfo-2.7.4-17.ns7.x86_64.rpm 
- nut-devel-2.7.4-17.ns7.x86_64.rpm	 
- nut-xml-2.7.4-17.ns7.x86_64.rpm

**Comment by gsanchietti** [comment]
**Test cases**
- [ ] Install nethserver-nut on NS 7.4, upgrade and check all RPMs dependency are resolved
- [ ] Update to NS 7.5 or install from ISO, at boot shorewall must *not* display:
       > kernel: ipt_ULOG: ULOG: fail to register logger.
- [ ] Check TLS policy is set to `20180330`
- [ ] The `/etc/fstab` file is not a template anymore, installing nethserver-samba shouldn't raise any error nor executing a filesystem remount
- [ ] The Server Manager must have session timeouts configured
- [ ] Windows file server page must have "`Grant full control to the creator` selected by default

**Comment by gsanchietti** [comment]
In `7.5.1804/base`:
- nethserver-httpd-3.2.1-1.1.g5664bfc.ns7.noarch.rpm
- nethserver-httpd-proxypass-3.2.1-1.1.g5664bfc.ns7.noarch.rpm
- nethserver-httpd-virtualhosts-3.2.1-1.1.g5664bfc.ns7.noarch.rpm

**Comment by DavidePrincipi** [comment]
**Additional test case**

- [ ] When upgrading to Rspamd, MxRecordStatus prop is deleted
- [ ] If MxRecordStatusProp was `enabled` (old default) `self` records are created in hosts DB
- [ ] In a gateway setup, LAN clients can connect to external SMTP servers on port 25

**Comment by nethbot** [comment]
in `7.5.1804/testing`:
- [nethserver-samba-4.0.1-1.4.g8f77faf.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/testing/x86_64/Packages/nethserver-samba-4.0.1-1.4.g8f77faf.ns7.noarch.rpm)

**Comment by gsanchietti** [comment]
Deleted RPMS:
.- nethforge/Source/SPackages/nethserver-webvirtmgr-1.1.1-1.ns7.src.rpm
- nethforge/Source/SPackages/webvirtmgr-4.8.9-1.192.ge1c7abb.ns7.src.rpm
- nethforge/x86_64/Packages/nethserver-webvirtmgr-1.1.1-1.ns7.noarch.rpm
- nethforge/x86_64/Packages/webvirtmgr-4.8.9-1.192.ge1c7abb.ns7.noarch.rpm
- base/Source/SPackages/nethserver-libvirt-1.1.0-1.ns7.src.rpm
- base/x86_64/Packages/nethserver-libvirt-1.1.0-1.ns7.noarch.rpm

**Comment by nethbot** [comment]
in `7.5.1804/testing`:
- [nethserver-firewall-base-3.3.2-1.21.gc8af8b9.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/testing/x86_64/Packages/nethserver-firewall-base-3.3.2-1.21.gc8af8b9.ns7.noarch.rpm)
- [nethserver-firewall-base-ui-3.3.2-1.21.gc8af8b9.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/testing/x86_64/Packages/nethserver-firewall-base-ui-3.3.2-1.21.gc8af8b9.ns7.noarch.rpm)

**Comment by gsanchietti** [comment]
Dahdi linux rebuild for new kernel, published in 7.5.1804/base:
- kmod-dahdi-linux-2.11.1-3.10.0_862.el7.3.ns7.x86_64.rpm
- dahdi-linux-kmod-debuginfo-2.11.1-3.10.0_862.el7.3.ns7.x86_64.rpm

**Comment by nethbot** [comment]
in `7.5.1804/updates`:
- [nethserver-base-3.2.0-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-base-3.2.0-1.ns7.noarch.rpm)

**Comment by nethbot** [comment]
in `7.5.1804/updates`:
- [nethserver-samba-4.1.0-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-samba-4.1.0-1.ns7.noarch.rpm)

**Comment by nethbot** [comment]
in `7.5.1804/updates`:
- [nethserver-httpd-admin-2.3.0-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-httpd-admin-2.3.0-1.ns7.noarch.rpm)

**Comment by nethbot** [comment]
in `7.5.1804/updates`:
- [nethserver-mail2-common-2.2.0-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-mail2-common-2.2.0-1.ns7.noarch.rpm)
- [nethserver-mail2-disclaimer-2.2.0-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-mail2-disclaimer-2.2.0-1.ns7.noarch.rpm)
- [nethserver-mail2-filter-2.2.0-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-mail2-filter-2.2.0-1.ns7.noarch.rpm)
- [nethserver-mail2-getmail-2.2.0-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-mail2-getmail-2.2.0-1.ns7.noarch.rpm)
- [nethserver-mail2-ipaccess-2.2.0-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-mail2-ipaccess-2.2.0-1.ns7.noarch.rpm)
- [nethserver-mail2-p3scan-2.2.0-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-mail2-p3scan-2.2.0-1.ns7.noarch.rpm)
- [nethserver-mail2-server-2.2.0-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-mail2-server-2.2.0-1.ns7.noarch.rpm)

**Comment by nethbot** [comment]
in `7.5.1804/updates`:
- [nethserver-httpd-3.2.2-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-httpd-3.2.2-1.ns7.noarch.rpm)
- [nethserver-httpd-proxypass-3.2.2-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-httpd-proxypass-3.2.2-1.ns7.noarch.rpm)
- [nethserver-httpd-virtualhosts-3.2.2-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-httpd-virtualhosts-3.2.2-1.ns7.noarch.rpm)

**Comment by nethbot** [comment]
in `7.5.1804/testing`:
- [nethserver-firewall-base-3.3.2-1.22.gda839d2.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/testing/x86_64/Packages/nethserver-firewall-base-3.3.2-1.22.gda839d2.ns7.noarch.rpm)
- [nethserver-firewall-base-ui-3.3.2-1.22.gda839d2.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/testing/x86_64/Packages/nethserver-firewall-base-ui-3.3.2-1.22.gda839d2.ns7.noarch.rpm)

**Comment by nethbot** [comment]
in `7.5.1804/testing`:
- [nethserver-firewall-base-3.3.2-1.23.g7df3f49.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/testing/x86_64/Packages/nethserver-firewall-base-3.3.2-1.23.g7df3f49.ns7.noarch.rpm)
- [nethserver-firewall-base-ui-3.3.2-1.23.g7df3f49.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/testing/x86_64/Packages/nethserver-firewall-base-ui-3.3.2-1.23.g7df3f49.ns7.noarch.rpm)

**Comment by nethbot** [comment]
in `7.5.1804/updates`:
- [nethserver-firewall-base-3.4.0-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-firewall-base-3.4.0-1.ns7.noarch.rpm)
- [nethserver-firewall-base-ui-3.4.0-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-firewall-base-ui-3.4.0-1.ns7.noarch.rpm)

**Comment by nethbot** [comment]
in `7.5.1804/updates`:
- [nethserver-nethforge-release-7-1.ns7.noarch.rpm](http://packages.nethserver.org/nethserver/7.5.1804/updates/x86_64/Packages/nethserver-nethforge-release-7-1.ns7.noarch.rpm)

**Comment by DavidePrincipi** [comment]
In `7.5.1804/updates`
nethserver-release-7-6.ns7.noarch.rpm

**Comment by gsanchietti** [comment]
In `7.5.1804/base`:
- nethserver-backup-config-2.1.0-1.ns7.noarch.rpm
